tests/integration-tests/webhooks/webhooks-test.js

```diff
@@ -0,0 +1,103 @@
+const { expect } = require('chai');
+const { WEBHOOK_EVENT_TYPES, EVENT_FORMAT_TYPE_JSON } = require('../../../src/common/consts');
+
+const webhookRequestSender = require('./helpers/requestCreator');
+
+describe('Webhooks api', function () {
+    this.timeout(5000000);
+    before(async function () {
+        await webhookRequestSender.init();
+    });
+
+    describe('Good requests', async function () {
+        describe('GET /v1/webhooks', async function () {
+            it('return 10 webhooks', async function() {
+                const webhooksToInsert = (new Array(10)).fill(0, 0, 10).map(index => generateWebhook(`My webhook ${index}`));
+                await Promise.all(webhooksToInsert.map(webhook => webhookRequestSender.createWebhook(webhook)));
+
+                const webhooksGetResponse = await webhookRequestSender.getWebhooks();
+                expect(webhooksGetResponse.statusCode).to.equal(200);
+
+                const webhooks = webhooksGetResponse.body;
+                expect(webhooks).to.be.an('array').and.have.lengthOf(10);
+            });
+        });
+        describe('POST /v1/webhooks', function () {
+            it('Create webhook and response 201 status code', async function() {
```

Comment:
add a test for create webhook with `global: true` since `global: false` is default